utils/lpac/files/lpac.sh

```diff
@@ -30,6 +30,11 @@ elif [ "$APDU_BACKEND" = "uqmi" ]; then
     UQMI_DEBUG="$(uci_get lpac uqmi debug 0)"
     export LPAC_QMI_DEV="$UQMI_DEV"
     export LPAC_QMI_DEBUG="$UQMI_DEBUG"
+elif [ "$APDU_BACKEND" = "mbim" ]; then
+    MBIM_DEVICE="$(uci_get lpac mbim device /dev/cdc-wdm0)"
+    MBIM_PROXY="$(uci_get lpac mbim proxy 0)"
```

Comment:
Why not enable the proxy by default?

It allows using MBIM via MM or mbim utils along LPAC